BrowserKit/Sources/ContentBlockingGenerator/ContentBlockerParser.swift

```diff
@@ -13,18 +14,29 @@ protocol ContentBlockerParser {
 class DefaultContentBlockerParser: ContentBlockerParser {
     // Key is each resource of an entity, so each properties for an entity's resource is easily accessible
     private var entities = [String: Entity]()
+    private let logger: Logger
+
+    init(logger: Logger = DefaultLogger.shared) {
+        self.logger = logger
+    }
 
     func parseEntityList(_ entitiesList: [String: Any]) {
         guard let entitiesRaw = entitiesList["entities"] as? [String: Any] else {
             return
         }
 
-        entitiesRaw.forEach {
-            let properties = ($0.value as! [String: [String]])["properties"]!
-            let resources = ($0.value as! [String: [String]])["resources"]!
-            let entity = Entity(properties: properties, resources: resources)
-            resources.forEach {
-                entities[$0] = entity
+        entitiesRaw.forEach { entitiesRawItem in
+            if let entitiesDict = entitiesRawItem.value as? [String: [String]],
+               let properties = entitiesDict["properties"],
+               let resources = entitiesDict["resources"] {
+                let entity = Entity(properties: properties, resources: resources)
+                resources.forEach {
+                    entities[$0] = entity
+                }
+            } else {
+                logger.log("Failed to parse entity for key: \(entitiesRawItem.key)",
```

Comment:
I think is good to have logs, but in those cases since we have already a lot, those i believe can be avoided.